SD DB Setup.sql

```diff
@@ -422,7 +422,7 @@ create table voting_rights
     voting_notes text,
     pending_leg text,
     date_modified timestamp not null default current_timestamp(),
-    user_modified varchar(20) not null);
+    user_modified varchar(20) default null);
     
```

Comment:
It should not be null. If it is, that is an error. The trigger should always populate it on insert and update.